Source/WebCore/page/scrolling/ScrollingTreeScrollingNode.cpp

```diff
@@ -142,7 +142,9 @@ bool ScrollingTreeScrollingNode::shouldRubberBandOnSide(BoxSide side, RectEdges<
     if (!pinnedEdges[side])
         return false;
 
-    if (isRootNode() && !scrollingTree()->clientAllowsMainFrameRubberBandingOnSide(side))
+    auto allowsMainFrameRubberBanding = scrollingTree()->clientAllowsMainFrameRubberBandingOnSide(side);
```

Comment:
The `auto` hides the fact that the value isn't a bool, making "allows" confusing. Call it `mainFrameRubberBandingBehavior`.